crates/fire_dynamics_tools/src/chapter_9/equation_9_4.rs

```diff
@@ -0,0 +1,27 @@
+pub fn effective_diameter(a_f: f64) -> f64 {
```

Comment:
It is `a_c` in the examples
```suggestion
pub fn effective_diameter(a_c: f64) -> f64 {
```

Reply:
It's shown as A_f in the equation:

<img width="971" height="262" alt="image" src="https://github.com/user-attachments/assets/013522a5-f3eb-4ac9-8a3a-17f97550b334" />

Reply:
It seems he started calling it area of combustible fuel later on, but I would leave it as it appears when the equation is defined.

Reply:
Yeh, this is again because we were looking at the equation in different places. Lets go with yours.